kong/db/strategies/off/init.lua

```diff
@@ -176,7 +188,7 @@ local function select(self, pk, options)
   local ws_id = workspace_id(schema, options)
   local pk = pk_string(schema, pk)
   local key = item_key(schema.name, ws_id, pk)
-  return select_by_key(schema, key, false)
+  return select_by_key(schema, key, need_follow(ws_id))
```

Comment:
if ws_id equals "*", i think we dont need to follow it. because you could select it with key directly

Reply:
No, the global key stores idx_value, not the serialized value.